visualization/app/codeCharta/ui/searchPanelModeSelector/searchPanelModeSelector.component.html

```diff
@@ -7,20 +7,12 @@
 		title="Folder structure"
 		><i class="fa fa-sitemap"></i
 	></md-button>
-	<md-button
-		class="state-selector-button middle"
-		ng-class="{ current: $ctrl._viewModel.searchPanelMode == 'flatten' }"
-		ng-click="$ctrl.onToggleSearchPanelMode('flatten')"
-		title="Flattened buildings"
-		><i class="fa fa-eye-slash"></i>
-		<div class="list-length-count" ng-show="$ctrl._viewModel.flattenListLength > 0"></div
-	></md-button>
 	<md-button
 		class="state-selector-button right"
-		ng-class="{ current: $ctrl._viewModel.searchPanelMode == 'exclude' }"
-		ng-click="$ctrl.onToggleSearchPanelMode('exclude')"
-		title="Excluded buildings"
-		><i class="fa fa-ban"></i>
-		<div class="list-length-count" ng-show="$ctrl._viewModel.excludeListLength > 0"></div
+		ng-class="{ current: ($ctrl._viewModel.searchPanelMode == 'blacklist') || ($ctrl._viewModel.searchPanelMode == 'exclude') }"
```

Comment:
```suggestion
		ng-class="{ current: ($ctrl._viewModel.searchPanelMode == 'blacklist') }"
```

I guess the former statement was needed while working on the code but not anymore?